cnf-tests/Dockerfile

```diff
@@ -61,6 +59,9 @@ RUN yum install -y numactl-devel make gcc && \
     cp hwlatdetect /hwlatdetect && \
     cp cyclictest /cyclictest
 
+FROM quay.io/openshift/origin-oc-rpms:4.13 AS oc
```

Comment:
can you bump the oc version?
btw we should have this container also under `registry.ci.openshift.org` no?

Reply:
sure, will do.
we have the go-builder under `registry.ci.openshift.org` because this is the only image created in redhat.
as for origin-oc-rpm version 4.14 I will check.